src/GraphQL/Types/Collections/IGraphTypeMappingProvider.cs

```diff
@@ -12,5 +12,5 @@ public interface IGraphTypeMappingProvider
     /// <param name="clrType">The CLR type to be mapped.</param>
     /// <param name="isInputType">Indicates whether the type is an input type.</param>
     /// <param name="preferredGraphType">The graph type that is suggested for this CLR type.</param>
-    Type? GetGraphTypeFromClrType(Type clrType, bool isInputType, Type? preferredGraphType);
+    Type? GetGraphTypeFromClrType([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties | DynamicallyAccessedMemberTypes.PublicMethods)] Type clrType, bool isInputType, Type? preferredGraphType);
```

Comment:
This is incorrect.  Simply having a CLR mapping to a graph type does not require all public methods/properties to be rooted.  Calls to `Field` will provide a static reference to the properties needed.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/3519#issuecomment-1412933180

How to deal with warning here?